src/config/templates.toml

```diff
@@ -20,27 +20,24 @@ label(if(current_working_copy, "working_copy"),
     git_head,
     format_short_commit_id(commit_id),
     if(conflict, label("conflict", "conflict")),
-  )
-  "\n"
+  ) "\n",
   separate(" ",
     if(empty, label("empty", "(empty)")),
     if(description, description.first_line(), description_placeholder),
-  )
-  "\n"
-)
+  ) "\n",
+))
```

Comment:
A minor downside of this is that it becomes a little difficult to match parentheses. The clear indentation helps.

Unfortunately, even editors that nicely color matching parentheses usually don't do it inside strings.

Reply:
Ok, moved `concat()` to separate line.